### PR TITLE
[WIP]Cleaning scripts under 'scripts/in_container'

### PR DESCRIPTION
**Pull request by OmairK**
Part of #10576 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
I think there are few "outside" containers methods that need to be modified, but the direction is good :)

**Comment by potiuk** [comment]
See the failing tests

**Comment by OmairK** [comment]
> See the failing tests
 
@potiuk I just made this PR just in case if someone else also starts working on this issue. I haven't tested it yet. Kind of wrapping my head around bash scripting :smile:

**Comment by potiuk** [comment]
> @potiuk I just made this PR just in case if someone else also starts working on this issue. I haven't tested it yet. Kind of wrapping my head around bash scripting 😄

If you need any help or advice - let me know :). Happy to help with my 20+years of bash experience (though every few months I learn something new :D)

**Comment by OmairK** [comment]
> If you need any help or advice - let me know :). Happy to help with my 20+years of bash experience (though every few months I learn something new :D)

Of course, thanks a lot :smile:

File: tests/bats/test_in_container_utils.bats [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# @test "test not in_container" {
+#     run container_utils::assert_in_container
+#     assert [ $status -eq 1 ]
+# }
+#
+# @test "test install_airflow" {
+#     container_utils::install_released_airflow_version "1.10.2"
+#     airflow_version=$(airflow version)
+#     assert [ airflow_version=="1.10.2" ]
+# }
```
Comment (werbolis) [inline tests/bats/test_in_container_utils.bats:45]: @OmairK did you aim to run these tests in the local dev env? The pre-commit hook uses `breeze static-check bats-tests`, which runs them inside in a container, so I'm pretty sure we can't have a "not in container" test here.

`test install_airflow` - the installation takes ages... Failure to install airflow should be easy to detect when building CI image. Perhaps we could stay safe and quick by stubbing pip and checking if it gets called with proper args? (e.g. https://github.com/jasonkarns/bats-mock)

I like the idea of testing these scripts, we definitely should have more tests! This PR is already somewhat bloated - if you're OK with that, I'd rather stick to two first tests in this file and add more in a separate PR.

**Comment by werbolis** [comment]
@potiuk I've rebased @OmairK 's work to current master and made some improvements to the utils script. Could you take a look? Most of changes outside of the utils script are prefix additions, but some (incomplete) refactoring was done by the way; hopefully it would make further fixing (in a series of small PRs) easier.

**Pull request closed without merging**